dist/av-core.js

```diff
@@ -6983,8 +6983,85 @@ module.exports = function(AV) {
         });
       })._thenRunCallbacks(options, this);
     },
+    /**
+     * Sign up or logs in a user with a third party access token.
+     * On success, this saves the session to disk, so you can retrieve the currently
+     * logged in user using <code>current</code>.
+     *
+     * <p>Calls options.success or options.error on completion.</p>
+     * 
+     * @param {String} platform Available platform for sign up. 
+     * @param {Object} data The response json data returned from third party token.
+     * @param {Object} [callback] An object that has an optional success function, that takes no arguments and will be called on a successful puSH. and an error function that takes a AV.Error and will be called if the push failed.
+     * @return {AV.Promise} A promise that is fulfilled with the user when
+     *     the login completes.
+     * @example AV.User.signUpOrlogInWithAccessToken(platform, data, {
+         *          success: function(user) {
+         *              //Access user here
+         *          },
+         *          error: function(error) {
+         *              //console.log("error: ", error);
+         *          }
+         *      });
+     * @see {@link https://leancloud.cn/docs/js_guide.html#绑定第三方平台账户}
+     */
+    signUpOrlogInWithAccessToken: function (platform, data, callback) {
+        /**
+         * Construct accessToken
+         */
+        var accessToken = {
+            authData: {}
+        }
+        accessToken.authData[platform] = data;
+        return this._logInWith(platform, {
+            "authData": data,
```

Comment:
老师你好，我重新看了下，用_login方法，貌似整个 accessToken 都可以不用了。
原版代码是这样的，使用了RESTAPI来Login。
后来最近发现文档里有了 _login，就可以省去 accessToken 啦。

    /**
     * Create a new anonymous user via REST API <br>
     * Can be integrated with REST API login <br>
     * @func AV.User.signUpOrloginWithAccessToken
     * @param {String} platform Available platform for sign up.
     * @param {Object} data The response data returned from third party token.
     * @param {Object} [callback] An object that has an optional success function, that takes no arguments and will be called on a successful puSH. and an error function that takes a AV.Error and will be called if the push failed.
     * @example AV.User.signUpOrloginWithAccessToken(platform, data, {
         *          success: function(user) {
         *              //Access user here
         *          },
         *          error: function(error) {
         *              //console.log(error.message);
         *          }
         *      });
     * @see {@link https://leancloud.cn/docs/js_guide.html#绑定第三方平台账户}
     */
    AV.User.signUpOrloginWithAccessToken = function (platform, data, callback) {
        /**
         * Construct accessToken
         */
        var accessToken = {
            authData: {}
        }
        accessToken.authData[platform] = data;
        /**
         * Custom REST method <br>
         * Tested to be working <br>
         * After REST login, current user can still be accessed by AV.User.current() <br>
         * Use in Shelf 2.0 REST methods
         */
        var url = "https://api.leancloud.cn/1.1/users";
        var xmlhttp = new XMLHttpRequest();
        xmlhttp.onreadystatechange = function () {
           if (xmlhttp.readyState == 4 && (xmlhttp.status == 200 || xmlhttp.status == 201)) {
               /**
                * result(User) will be a JS Object
                * key: username
                * key: sessionToken
                * key: createdAt
                * key: objectId
                */
               var myArr = JSON.parse(xmlhttp.responseText);
               //Check for undefined, preventing the rest of code to crash
               var sessionToken = myArr.sessionToken;
               AV.User.become(sessionToken, {
                   success: function (u) {
                       callback.success(u);
                   },
                   error: function (error) {
                       callback.error(error);
                   }
               });
           } else if (xmlhttp.readyState == 4 && xmlhttp.status != 201 && xmlhttp.status != 200) {
               //Check for undefined, preventing the rest of code to crash
               callback.error({
                   "message": "state: " + xmlhttp.status + " | " + xmlhttp.readyState
               });
           }
        }
        xmlhttp.open('POST', url, true);
        xmlhttp.setRequestHeader("Content-Type", "application/json");
        /**
        * Use LC-Sign, require md5
        */
        xmlhttp.setRequestHeader("X-LC-Id", APP_ID);
        xmlhttp.setRequestHeader("X-LC-Key", APP_KEY);
        xmlhttp.send(JSON.stringify(accessToken));
        /**
         * LC official support since v0.6.4
         * Tested to be working with anonymouos, Facebook
         */
        AV.User._logInWith(platform, {
            "authData": data,
            success: function (user) {
                callback.success(user);
            },
            error: function (error) {
                callback.error(error);
            }
        });
    }